src/smk/tool/baseMaps/tool-baseMaps.js

```diff
@@ -57,7 +57,10 @@ include.module( 'tool-baseMaps', [
                             zoomSnap: 0
                         } );
 
-                        var bmLayers = smk.$viewer.createBasemapLayer( bm.id )
+                        m.createPane('basemaps');
+                        m.getPane('basemaps').style.zIndex = 100;
+
+                        var bmLayers = smk.$viewer.createBasemapLayer( smk.viewer.esriApiKey, bm.id )
```

Comment:
I'm not sure we need to read the esriApiKey value from the smk.viewer config, only to pass it back to the viewer object. It might be a little cleaner/maintainable if we only access the key where it is needed (ie. only within `createBasemapEsri` in viewer-leaflet.js).

Reply:
The reason it's passed around is that we only have a reference to the "smk" object in the init method, and then pass it from init to where it's needed. 

We might be able to do this using the SMK global variable. I see there's SMK.CONFIG.Viewer, which includes the "basemap" property, but it looks like I'd need to add the "esriApiKey" property. Is this what you were thinking, or do you know of a better way to get the property from within the createBasemapEsri() method?

Reply:
I looked into this further ... it looks like the values in SMK.CONFIG don't get updated as the app is initialized and run; they remain as set in smk.js. For example, SMK.CONFIG.viewer.baseMap is set as "Topographic" and shows that value when debugging, regardless of the basemap set in smk-config.js.

Reply:
The config is available on the global SMK object at `SMK.MAP[0].viewer`. The reason I mention it is that we end up passing around the key when it is not needed (eg. for the StamenTonerLight basemap). Though I suppose we also pass an unused id into `basemap.create()` when the basemap is StamenTonerLight. I'll leave the choice up to you.